tests/utils.py

```diff
@@ -64,6 +64,9 @@ def patch_workbench(self):
             lambda _, html: re.sub(r'"/static/([^"]*)"', r'"/course/test-course/assets/\1"', html),
             create=True,
         )
+        self.apply_patch(
+            'drag_and_drop_v2.drag_and_drop_v2.get_grading_ignore_decoys_waffle_flag'
+        )
```

Comment:
This condition will be truthy (`Mock != False`). We may want something like this instead:
```suggestion
        self.apply_patch(
            'drag_and_drop_v2.drag_and_drop_v2.get_grading_ignore_decoys_waffle_flag',
            lambda: Mock(is_enabled=lambda _: False),
        )
```